roles/README.adoc

```diff
@@ -1,17 +1,9 @@
-= Ansible Good Practices - AGP
+= Roles Good Practices
 
 This document is intended to be a standards document for the Ansible roles contained in the
 AGP Roles repositories.
```

Comment:
To match the main page above, which abbreviates Good Practices for Ansible as GPA:
```suggestion
GPA Roles repositories.
```

Reply:
I removed it altogether, the introduction anyway didn't fit the context.